NewSpring/corinth
# chore: Hide campaign if it doesn't exist

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR removes the campaign item from the feed if there isn't one.

![Screenshot 2020-06-16 09 08 17](https://user-images.githubusercontent.com/3229463/84782273-d07cc280-afb5-11ea-9999-d3de7ae3b39f.png)

### How do I test this PR?
Remove all associated items from the Campaign content channel entry and load the app. You shouldn't see a campaign item at all. 

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
